docs/api-reference/server/overview-middleware.md

```diff
@@ -81,28 +81,28 @@ Called when a connection is closed.
 
 ##### onStart(message)
 
-Called when a `xviz/start` message is inititiated
+Called when a `XVIZ/START` message is inititiated
```

Comment:
Can we keep these lower case?  This is part of the wire protocol which I consider different than an enum.

Reply:
Sure.  I don't see these even defined in the `xviz/v2` protobuf definitions actually.  Is that intentional?

Reply:
I think I may have been conflating two separate ideas. The wire protocol, here as `xviz/start` which represents the message type `start`.

I have been treating them as basically the same and given the same treatment.  It sounds like the wire protocol is separate.